### PR TITLE
resolved issue #2383 

### LINKED ISSUE #2383: Add validation for max dosage in PRN prescription
image

Add backend validation to check that max dosage in 24 is greater than or equal to base dosage

https://github.com/coronasafe/care/blob/68a3a8281cf72eb859b5951ffbadda0399c3010f/care/facility/api/serializers/prescription.py#L97-L151

### PR DESCRIPTION
**Pull request by Dishaswamy**
resolved #2383 

@ohcnetwork/care-backend-maintainers @ohcnetwork/care-backend-admins

### CODE REVIEW

File: care/facility/api/serializers/prescription.py [hunk 1]
```diff
@@ -94,59 +94,71 @@ class Meta:
             "is_migrated",
         )
 
-    def validate(self, attrs):
-        if "medicine" in attrs:
-            attrs["medicine"] = get_object_or_404(
-                MedibaseMedicine, external_id=attrs["medicine"]
+   def validate(self, attrs):
```
Comment (anroopak) [inline care/facility/api/serializers/prescription.py:97]: Please add valid test cases.

File: config/authentication.py
Comment (anroopak) [inline config/authentication.py:1]: Changes in this file is not related to the mentioned issue. Please isolate this to a seperate PR.

**Review by anroopak — CHANGES_REQUESTED** [review]

**Comment by sainak** [comment]
hey @Dishaswamy this prs has unrelated changes to files and the indentation is also incorrect,
read the following docs and open a new pr again:
https://docs.ohc.network/docs/contributing/
https://github.com/ohcnetwork/care/blob/develop/CONTRIBUTING.md

**Comment by sainak** [comment]
You should start working on a Pull request only when you are assigned the issue.

finish up https://github.com/ohcnetwork/care/pull/2450 first

**Pull request closed without merging**